test/unit/tasks/impersonation_admin_user_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Tasks::ImpersonationAdminUserTest < ActiveSupport::TestCase
+  setup do
+    3.times do
+      FactoryGirl.create(:active_admin,
+                            username: '3scaleadmin',
+                            account: FactoryGirl.create(:simple_provider)
+                          )
+    end
+    FactoryGirl.create_list(:active_admin, 2)
+  end
+
+  test 'update' do
+    execute_rake_task 'impersonation_admin_user.rake', 'impersonation_admin_user:update', 'example-username', 'domain.example.com'
+
+    users = User.where(username: 'example-username')
+    assert_equal 0, User.where(username: '3scaleadmin').count
+    assert_equal 3, users.count
```

Comment:
Maybe worth verifying it changes only `3scaleadmin` users?
If you'd remove `where(username: '3scaleadmin')` then this test would still pass, no?

Reply:
It wouldn't pass for this:
https://github.com/3scale/porta/pull/222/files#diff-b1be6eb9ffabbfbde3cc775e1976a2b4R13

Reply:
I see, so instead of ensuring there are 2 old left it ensures there are just 3 new. It could be more obvious somehow.

Reply:
ahaa 👍 I've just added `assert_equal 2, User.where('username <> \'3scaleadmin\'').where('username <> \'example-username\'').count` as well 😄